### ISSUE TITLE
OMBI v4 Not Processing TV Show Requests To SONARR v4

### ISSUE THREAD

**Issue by EGeekBMe** [Ombi-app/Ombi#4054]
**Describe the bug**
Requests made via OMBI v4 are not being added to SONARR v4 and are hanging at "Processing Request" in OMBI and it never clears or gets added to SONARR.

I want to note that all requests for Movies submitted via OMBI get added to RADARR v4 just fine and without any kind of issue.

**To Reproduce**
1. Request a TV Show via OMBI
2. Manually Approve Request if Auto Approve is not enabled
3. Request just sits with "Processing Request" status

**Expected behavior**
I expect the requested content to show up on SONARR.

**Screenshots**
![image](https://user-images.githubusercontent.com/58891608/107996599-fb347600-6f9d-11eb-81de-50662a12ecc9.png)

**Logs (Logs directory where Ombi is located)**
I am not sure if the logs show any errors but here is the GIST of the log: https://gist.github.com/EGeekBMe/e39ea1f17feeb29d5dfae5f481b95847

**Desktop (please complete the following information):**
 - OS: Linux 5.4.31-gentoo-whatbox (seedbox)

**Ombi Version (please complete the following information):**

- Version: 4.0.1035.0
- Media Server: Plex
- Database Type: SQLite

**Additional context**
Things that I have tried...
- The checkbox for Sonarr is set to ENABLED.
- Test Connectivity on both shows as successful.
- Base User has Auto Approve permissions for both TV and MOVIE (Movies work fine)
- Tried as admin user and same result.
- C&P the SONARR API
- Confirmed port
- No Base URL in use but tried with just a / which passes connectivity test.
- Also tried with base url of /sonarr and fails connectivity test.

**Comment by EGeekBMe** [comment]
I've searched for this issue many times and have found several from long times ago and nothing really recent. So if there is something for this issue that is more recent that I can piggy back on versus creating this new item, please let me know where that may be and I'll be more than happy to head over there. Thank you all in advance. :)

**Comment by tidusjar** [comment]
If a request is Processing Request that means that it's processing... That it's In sonarr and not yet available. 

You sure you have enabled sonarr in your settings?

**Comment by EGeekBMe** [comment]
Yes, I have enabled Sonarr in the settings.

It wasn't in Sonarr as it was not showing up in the listing of shows in Sonarr. I entered this show in manually to Sonarr and did a search on season one and nothing was showing up as available due to unwanted formats or not enough seeders.

I adjusted the indexers to require fewer seeders and then there were options available. I deleted that show out of Sonarr, deleted the request from Ombi. I then resubmitted the request via Ombi and it became available in Sonarr and started downloading episodes.

I never thought to look at the indexer requirements because the show was most certainly not showing in Sonarr. My default filter is shows with missing episodes and it was not showing there at all. Then when I went in to search for that show in my listings, it was not appearing at all.

So not sure why it wouldn't appear in my Sonarr list just because it couldn't find any positive results in a search but that doesn't makes sense to me.

I guess this can be closed.

Thanks.
